server/src/utils/expenses.ts

```diff
@@ -49,7 +49,6 @@ async function calculateTotalExpenseForBudget(budget: Budget): Promise<number> {
   }
 
   // Fetch expenses for the budget and return the total amount
-  // TODO: Fetch group expenses
```

Comment:
Another random question, we deleted this todo bc we are implementing grabbing all the group expenses on a different call, right(?)

Reply:
When I wrote this TODO, I thought that we'd store paid group expenses and individual expenses in different tables (`GroupExpenseSplit`, and `Expense`), but after implementing group expenses I made it so that when a group expense is marked as paid by a user, it creates a new `Expense`. So, all the expenses for a user (both individual and group) are in the same table, so we no longer need to have a seperate fetch for paid group expenses.